charts/karmada-operator/crds/operator.karmada.io_karmadas.yaml

```diff
@@ -70,7 +70,7 @@ spec:
                           component. In the future, we will provide a more structured
                           way to configure the component. Once that is done, this
                           field will be discouraged to be used. Incorrect settings
-                          on this feild maybe lead to the corresponding component
+                          on this field maybe lead to the corresponding component
```

Comment:
this issue has been fixed on the master. You might need to rebase your code against master and force-push again.

Reply:
Sure! will do it.

Reply:
Here are the steps for reference:
- git checkout operator-typo-fix
- git fetch upstream
- git rebase upstream/master
- git push origin -f

Reply:
@RainbowMango thanks a lot for the heads up! Now it is all good.